app/serializers/curated_list_serializer.rb

```diff
@@ -8,13 +8,10 @@ def to_json(*_args)
       {
         "name" => curated_list.name,
         "description" => curated_list.description,
-        "slug" => curated_list.slug
-        # "articles" => curated_list.articles.map do |article|
-        #   {
-        #     "title" => article.title,
-        #     "slug" => article.slug
-        #   }
-        # end
+        "slug" => curated_list.slug,
+        "articles" => curated_list.articles.map do |article| 
+          article.to_json
```

Comment:
Leaving a note here to myself to revisit this, I think there's a better way to include the articles as JS objects without pre-parsing them as JSON.